i3lock.c

```diff
@@ -1391,8 +1412,12 @@ int main(int argc, char *argv[]) {
         {"radius", required_argument, NULL, 402},
         {"ring-width", required_argument, NULL, 403},
 
-        // alignment
+		// pass keys
+        {"pass-media-keys", no_argument, NULL, 'm'},
+        {"pass-screen-keys", no_argument, NULL, 's'},
```

Comment:
s and p are both already used ("line-uses-inside" and "pointer" above) - it's fine to just leave them with a numeric value in the 900s to switch on.

Reply:
Those are using capital letters, no? I remember doing a search for those flags and they didn't come up.

Reply:
unfortunately not - check lines 1405 and 1375 above. ('s' and 'S' are both already in use)

There's generally too many args to reliably give meaningful short arg names to any new args - plus, most people just run i3lock through a script that holds all the args, so short args aren't really necessary anyhow.

Reply:
True. Have removed them all. 600s were available so I used those.

Reply:
Perfect. Thank you very much!

Reply:
Cool, no worries!